BUILD.md

```diff
@@ -59,9 +55,9 @@ Runs after the top-level `yarn install`. This ensures `./packages/plexus` builds
 
 Note that `./packages/plexus` does not yet have any tests, as tracked in issue [#340](https://github.com/jaegertracing/jaeger-ui/issues/340).
 
-`./packages/jaeger-ui` uses [Jest](https://jestjs.io/) for testing. It can be useful to directly run tests for that package by running `yarn test` from its directory, rather than the repository root.
+`./packages/jaeger-ui` uses [Jest](https://jestjs.io/) for testing. It can be useful to directly run tests for that package by running `npm run test` from its directory, rather than the repository root.
 
-Tests for React components in `./packages/jaeger-ui` make extensive use of Jest's [snapshot testing](https://jestjs.io/docs/28.x/snapshot-testing) functionality. These snapshots can be regenerated by running `yarn test -u` from the package directory to regenerate all snapshots, or `yarn test -u --testNamePattern <test name>` to regenerate snapshots for a subset of tests only.
+Tests for React components in `./packages/jaeger-ui` make extensive use of Jest's [snapshot testing](https://jestjs.io/docs/28.x/snapshot-testing) functionality. These snapshots can be regenerated by running `npm test -- -u` from the package directory to regenerate all snapshots, or `npm test -- -u --testNamePattern <test name>` to regenerate snapshots for a subset of tests only.
```

Comment:
can we provide an example from the existing tests? It's not clear what `<test name>` means, e.g. in this test file:
```
 PASS  src/utils/readJsonFile.test.js
  fileReader.readJsonFile
    ✓ rejects when given an invalid file (2 ms)
    ✓ does not throw when given an invalid file (1 ms)
    ✓ loads JSON data, successfully (8 ms)
    ✓ loads JSON data (OTLP), successfully (2 ms)
    ✓ rejects an OTLP trace (4 ms)
    ✓ rejects malformed JSON (1 ms)
    ✓ loads JSON-per-line data (1 ms)
```
Is it the actual title/description? Is "pattern" a reference to some regex?

Reply:
It's a regex matching the full test name, i.e. test name and all surrounding describe blocks: https://jestjs.io/docs/cli#--testnamepatternregex
I've updated the text and added a link to the docs.